weather.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+require("dotenv").config();
+const WEATHER_API_KEY = process.env.WEATHER_API_KEY;
+const axios = require("axios");
+
+// this should use find()
+let haveWeatherData = (city) => {
+  for (let data of weather) {
+    if (data.city_name.toLowerCase() === city) {
+      return data;
+    }
+  }
+}
+
+
+class Forecast {
+  constructor(date, description) {
+    this.date = date, 
+    this.description = description;
+  }
+}
+
+let getWeather = async (req, res) => {
+  const q = req.query;
+  const lat = q.lat;
+  const lon = q.lon;
+  const searchQuery = q.searchquery;
+  const dayData = [];
+
+  let weatherApiUrl = `https://api.weatherbit.io/v2.0/current?key=${WEATHER_API_KEY}&lat=${lat}&lon=${lon}&units=I`;
```

Comment:
For the `weatherApiUrl, same goes, changing this to `const` makes this a bit more clear and this will never be reassigned 👍